### PR TITLE
Added created and last_modified columns to family documents.

### PR DESCRIPTION
**Pull request by katybaulch**
# Description

- Added created and last modified columns to family document entity
- Added update trigger for family document table

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## How Has This Been Tested?

Family document tests updated in test_family.py.

## Reviewer Checklist

- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

**Review by diversemix — COMMENTED** [review]
All good - just the tests now :smile:

File: pyproject.toml [hunk 1]
```diff
@@ -46,6 +46,10 @@ pyright = "^1.1.294"
 ruff = "^0.0.291"
 types-SQLAlchemy = "^1.4.31"
 
+[tool.poetry.group.dev.dependencies]
+surrogate = "^0.1"
```
Comment (diversemix) [inline pyproject.toml:50]: This repo is not very well supported - not saying we shouldn't use it - but perhaps we should fork it into `climatepoicyradar` ?

Comment (katybaulch) [reply pyproject.toml:50]: I'll raise a ticket for this.

**Review by diversemix — APPROVED** [review]
Looks great - as your going with the other entities - it would be good to have their own tests (i.e. move from ingestion to a new `database` folder) So we can explicitly test the updates happen at the db layer - i.e. with `db.execute("UPDATE ...."

**Pull request merged**